meta-mbl-distro/classes/mbl-distro-image-development.bbclass

```diff
@@ -0,0 +1,31 @@
+# Copyright (c) 2019 Arm Limited and Contributors. All rights reserved.
+#
+# SPDX-License-Identifier: MIT
+
+###############################################################################
+# IMAGE_INSTALL:
+#   Specify the packages installed in the distribution images prior to
+#   inheriting from core-image to override the default behaviour.
+#
+#     packagegroup-mbl-production   production packages added for this image.
+#     packagegroup-mbl-development  development packages added for this image.
+#     CORE_IMAGE_EXTRA_INSTALL      Symbol conventionally defined in local.conf
+#                                   to add extra packages.
+#
+# IMAGE_FEATURES: specify additional packages
+#   debug-tweaks
+#       Included in image so root has empty password.The extrausers class
+#       in also used so EXTRA_USERS_PARAMS can specify the empty password.
+###############################################################################
```

Comment:
Actually, because we are adding `debug-tweaks` to the `IMAGE_FEATURES` we don't even need to set the root password to empty since it is already set in the `rootfs-postcommands.bbclass` when having `debug-tweaks`. Addressed in e0bad67.